alfredo-zimperz/lectures/tour-of-heroes-redux/src/redux/actions/actionCreators.js

```diff
@@ -0,0 +1,47 @@
+import actionTypes from './actionTypes';
+import HEROES from '../../constants/heroes.mock';
+
+export function loadHeroes() {
+  return {
+    type: actionTypes.LOAD_HEROES,
+  };
+}
+
+export function addHero(heroName) {
+  const nextId = HEROES.length;
```

Comment:
i think you will be adding the same id to your heroes.

Reply:
I agree, its creating always the same Id :/

Reply:
Fixed, thank you guys!